### PR TITLE
Sorts DESC_LORE and DESC_FLUFF

### PR DESCRIPTION
**Pull request by alsoandanswer**
Seperates out the jokes in desc_fluff from actual lore for consistency.

relies on https://github.com/Aurorastation/Aurora.3/pull/14836.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/atoms.dm [hunk 1]
```diff
@@ -30,11 +30,11 @@
 
 	var/gfi_layer_rotation = GFI_ROTATION_DEFAULT
 
-	// Extra descriptions.
-	var/desc_fluff = null // Text about the atom's fluff description, if any exists.
+	// Extra descriptions. // Regular descriptions typically reserved for dry, raw descriptions.
+	var/desc_fluff = null // Text about the atom's fluff description, if any exists. Reserved for funny jokes and snarky comments, mostly.
```
Comment (SleepyGemmy) [inline code/game/atoms.dm:34]: this is not true and is a subjective appropiation of the variable.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
(a paste from the previous PR since this does the same thing as 14836.)



jokes don't belong in the descriptions of items, since they're descriptions.
if something is more of a quip or humorous observation, then it could fit in desc_fluff (see the example below).
the reason a cultist description exists is because only cultists can do certain interactions with certain items, not every antagonist.

for reference:
desc is for shorthand descriptions of items. example: "A lightbulb made out of glass."
desc_fluff is for longer descriptions of items, extra information, and/or lore. example: "This particular lightbulb is manufactured by the Idris Incorporation for use in service installations across the galaxy. Commonly thought to be of cheap quality."
desc_antag is for antagonism descriptions.
desc_cult is for cultist related descriptions.

File: code/game/atoms.dm [hunk 2]
```diff
@@ -30,11 +30,11 @@
...
 	var/desc_info = null // Blue text (SPAN_NOTICE()), informing the user about how to use the item or about game controls.
 	var/desc_antag = null // Red text (SPAN_ALERT()), informing the user about how they can use an object to antagonize.
-	var/desc_cult = null // Purple text (SPAN_CULT()), telling the user, if they're a cultist, how they can use certain items related to being a cultist.
+	var/desc_lore = null // Purple text (SPAN_CULT()) that gives the atom's lore info, be it species, corporate or whatever.
```
Comment (NonQueueingMatt) [inline code/game/atoms.dm:37]: Not SPAN_CULT anymore.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by SleepyGemmy** [comment]
closing as PR 14843 was merged.

**Pull request closed without merging**